akka-stream/src/main/boilerplate/akka/stream/scaladsl/UnzipWithApply.scala.template

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2014-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.scaladsl
+
+import akka.stream._
+import akka.stream.impl.GenJunctions._
+import akka.stream.impl.StreamLayout
+
+trait UnzipWithApply {
+
+  [2..20#/**
+   * Create a new `UnzipWith` specialized for 1 outputs.
+   *
+   * @param unzipper unzipping-function from the input value to 1 output values
+   */
+  def apply[O, [#A1#]](unzipper: O ⇒ ([#A1#])): UnzipWith1[O, [#A1#]] = {
+    val shape = new FanOutShape1[O, [#A1#]]("UnzipWith1")
+    new UnzipWith1(shape, new UnzipWith1Module(shape, unzipper, OperationAttributes.name("UnzipWith1")))
+    }
+    #
+
+  ]
```

Comment:
Spent some time on figuring out how to run "Genjavadoc Java API documentation to" to check new solution. Tried genjavadoc:doc for stream project - does not work for me. Can you give me a hint?